ironfish/src/workerPool/tasks/createMinersFee.ts

```diff
@@ -23,23 +24,23 @@ export class CreateMinersFeeRequest extends WorkerMessage {
     const bw = bufio.write(this.getSize())
     bw.writeVarBytes(BigIntUtils.toBytesBE(this.amount))
     bw.writeVarString(this.memo, 'utf8')
-    bw.writeVarString(this.spendKey, 'utf8')
+    bw.writeVarBytes(this.spendKey)
```

Comment:
Maybe just `writeBytes` to make consistent with the below change `reader.readBytes(ACCOUNT_KEY_LENGTH)`

Reply:
FIxed